terracumber_config/tf_files/SUSEManager-Test-NUE.tf

```diff
@@ -81,8 +81,8 @@ provider "libvirt" {
 module "cucumber_testsuite" {
   source = "./modules/cucumber_testsuite"
 
-  product_version = "head"
-  
+  product_version = "4.0-released"
```

Comment:
You are mixing: `Manager-4.1` branch for cucumber, and `4.0-released` for the product version.

But besides that: are you sure 1000% sure this should be `released`?

Reply:
ah right, 4.1 was a typo, thank you.

I don't see any problem with released as I am testing a branch based on the last released MU. Do you see any problem?

Reply:
I am pretty sure that if you use it, you will only get `released` packages, without the packages you want to test. Check the sumaform doc.